libkbfs/journal_server.go

```diff
@@ -301,6 +455,11 @@ func (j *JournalServer) Disable(ctx context.Context, tlfID TlfID) (err error) {
 		return fmt.Errorf("Journal still has %d MD entries", length)
 	}
 
+	select {
+	case bundle.shutdownCh <- struct{}{}:
```

Comment:
This should be a `close()` IMO, since you don't want the shutdown write to get lost.